test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -413,34 +412,34 @@ object TestAspect extends TimeoutVariants {
    * An aspect that only runs a test if the specified environment variable
    * satisfies the specified assertion.
    */
-  def ifEnv(env: String, assertion: Assertion[String]): TestAspectAtLeastR[Has[Live] with Has[Annotations]] =
+  def ifEnv(env: String, assertion: String => Boolean): TestAspectAtLeastR[Has[Live] with Has[Annotations]] =
```

Comment:
I think now that this is no longer an assertion, it would be more idiomatic to take two parameter lists, e.g. `ifEnv("foo")(_.contains("bar"))`.

Reply:
Done!